static/js/forms.js

```diff
@@ -4,53 +4,25 @@ if (typeof forms !== 'undefined') {
 
 var forms = {}
 
-// Marketo form state/province select/optgroup js
 forms.stateFormField = function() {
-	var formStateLabel = document.querySelector('[for="State"]');
-  var formStateSelect = document.getElementById('State');
+  var countrySelect = document.getElementById('Country');
   var formStateContainer = document.querySelector('.mktoPlaceholderState');
-  var formUSAOpt = document.querySelector('[value="SelectState"]');
-  var formCanadaOpt = document.querySelector('[value="SelectProvince"]');
 
   // Only if things exist...
   if (formStateContainer === null) {
     return;
   }
 
-	// starting state, no field
-	state('default');
+  countrySelect.addEventListener('change', function(e) {
```

Comment:
Could you check this element exists aswell when you null check the formStateContainer

Reply:
Good shout - will do :+1: